src/pages/Lend/LoanChannelCategoryPage.vue

```diff
@@ -56,7 +56,7 @@ export default {
 		preFetch(config, client, args) {
 			return client.query({ query: uiConfigSettingQuery, variables: { key: CATEGORY_REDIRECT_KEY } })
 				.then(({ data }) => {
-					const query = args?.route?.query ?? {};
+					const query = args?.route?.value?.query ?? {};
```

Comment:
Do we instead need `args?.route?.value ?? args?.route`? We do something similar in other places.

Reply:
I don't see that approach in other places, also `route?.value` is a vue3 req.

Reply:
We use `const currentRoute = route.value ?? route ?? {};` throughout `ui`

Reply:
It's related to full refresh vs router, I think.